src/app/settings/settings-routing.module.ts

```diff
@@ -0,0 +1,22 @@
+import { NgModule } from '@angular/core'
+import { RouterModule, Routes } from '@angular/router'
+import { SettingsComponent } from './components/settings/settings.component'
+import { ClearCacheComponent } from './components/clear-cache/clear-cache.component'
+
+const routes: Routes = [
+  {
+    path: '',
+    component: SettingsComponent,
+    children: [
+      { path: '', redirectTo: 'action-triggers', pathMatch: 'full' },
+      { path: 'action-triggers', loadChildren: () => import('./action-triggers/action-triggers.module').then(module => module.ActionTriggersModule) },
```

Comment:
It would be nice if the various paths where extracted into an enum (take a look at `app-routing.module.ts`).
This way we can use the enum throughout our code when we need to navigate to a path and if we ever need to change the path, we just need to update the value of our enum. 

Right now, the paths are magic strings, and if we want to update it, we have to find all magic strings for it throughout our codebase which is not ideal

Reply:
Done